components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.hpp

```diff
@@ -90,8 +61,7 @@ class Nudging : public AtmosphereProcess
   // Must add this here to make it public for CUDA
   // (refining) remapper vertically-weighted tendency application
   void apply_vert_cutoff_tendency(Field &base, const Field &next,
-                                  const Field &p_mid, const Real cutoff,
-                                  const Real dt);
+                                  const Field &p_mid, const Real dt) const;
```

Comment:
I don't feel strongly about this, and I can change it back.

Imho, if a function is a method, it makes no sense to pass a class member to it, unless you sometimes invoke it with different arguments, like sometimes `my_method(m_var1);` and sometimes `my_method(m_var2);`. But if you _always_ call it with just `m_var1`, it's seems clearer to not show it in the signature. Everything in the signature is something "configurable", something that can take many (2+) possible values. If it can only take one value, it's not a function argument.

Let's play rock-paper-scissor, and who wins gets to keep his version!

Reply:
Yeah, we can do that. It will also benefit the run method, where we would simply call `apply_tendency`. All in all, I think it would reduce the lines of code, improving readability and maintainability.

Reply:
Ok, I unified them in one fcn, and left the function "not pure"